### DISCUSSION TITLE
Elementor popup function for gravity forms

### DISCUSSION THREAD

**Discussion by GringoPost** [elementor/elementor#25263] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I am unable to use the **Elementor** popup window with **gravity forms**. My entire business depends on gravityforms submissions. There is a 3rd-party Elementor add-on but it doesn't support popup windows. I searched for weeks all over, and seeked help from pros and no one has a solution except switching over to a new web builder, and I am not about to do that. It's stressful to go through this. Could you please consider a fix for this nightmare?  

- [ ] 

### Describe the solution you'd like.

I would like to be able to use the **popup** windows with **gravityforms**. My entire business depends on **gravityforms** submissions. Here is test the link to the popup button gringopost.com/testing-form/

### Describe alternatives you've considered.

Switching to a different web builder. I can't do without the gravityforms, I have been using them for 10 years. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by GringoPost** [comment]
Please take a look and consider it.

**Reply by ThrudUK** [reply]
This was reported as a "bug" years ago... https://github.com/elementor/elementor/issues/7245?fbclid=IwZXh0bgNhZW0CMTAAAR3AYpw8i02wAuZnj9bFt-QJRZQ3ytctJ3OtexnfUqczrmTjyrlYUzhPJKw_aem_VR2lxzKAUwI9pwUt35pWVw

Nice to see it pop up as a feature request :)
